### DISCUSSION TITLE
Wrong REGION - Private internet access

### DISCUSSION THREAD

**Discussion by huydinh3010** [qdm12/gluetun#1221] [Q&A] [answered]
Im using latest version (commit 192a7a5)
Service provider: Private internet access
I configured SERVERS_REGIONS = Vietnam, but sometimes, the IP address I was given is Singapore IP address. 
Is there any way to make sure the IP addresses are always Vietnam IPs? 

Configurations:
![Untitled](https://user-images.githubusercontent.com/32333862/199150417-709c56f1-183d-482b-83c2-9d7a2b6c1868.png)

Logs:
![Untitled2](https://user-images.githubusercontent.com/32333862/199150392-14621ac3-de33-45a0-96ce-5b58123671a0.png)

**Comment by qdm12** [comment, marked as answer]
Ip data is fetched from ipinfo.io which sometimes has no longer valid information.

If you try `188.214.152.72` on another website such as https://ipdata.co/ it'll show vietnam.

I need to improve the ip data fetching mechanism soon, I'll see what I can do.
